### PR TITLE
Implement basic NowPlaying Command

### PR DESCRIPTION
**Pull request by OtterBoops**
Will format the rest of the embeds when I get a greenlight on the basic implementation of youtube nowplaying

### CODE REVIEW

File: src/main/kotlin/dev/arbjerg/ukulele/utils/TextUtils.kt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package dev.arbjerg.ukulele.utils
+
+import java.util.concurrent.TimeUnit
+
+class TextUtils {
+    companion object {
```
Comment (Frederikam) [inline src/main/kotlin/dev/arbjerg/ukulele/utils/TextUtils.kt:6]: Remove `companion object` and replace `class` with `object`.

https://kotlinlang.org/docs/tutorials/kotlin-for-py/objects-and-companion-objects.html

File: src/main/kotlin/dev/arbjerg/ukulele/command/NowPlayingCommand.kt [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    private fun getDefaultEmbed(): MessageEmbed {
+        val message = EmbedBuilder()
+        message.setTitle("Not yet implemented.")
+        return message.build()
+    }
+
+
+    override fun HelpContext.provideHelp() {
+        addUsage("")
+        addDescription("Displays general help. (unfinished)")
```
Comment (Frederikam) [inline src/main/kotlin/dev/arbjerg/ukulele/command/NowPlayingCommand.kt:62]: Please change

File: src/main/kotlin/dev/arbjerg/ukulele/command/NowPlayingCommand.kt [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+    override suspend fun CommandContext.invoke() {
+        if (player.tracks.isEmpty())
+            return reply("Not playing anything.")
+
+        replyEmbed(buildEmbed(player.tracks[0]))
+    }
+
+    private fun buildEmbed(track: AudioTrack): MessageEmbed {
+        return when(track.sourceManager.sourceName){
```
Comment (Frederikam) [inline src/main/kotlin/dev/arbjerg/ukulele/command/NowPlayingCommand.kt:24]: Try switching by track type 

https://github.com/Frederikam/FredBoat/blob/dev/FredBoat/src/main/java/fredboat/command/music/info/NowplayingCommand.kt#L65-L74

**Comment by OtterBoops** [comment]
Realized that `::list` breaks when there's a livestream in the queue, but I'll fix it outside of this PR.

**Review by Frederikam — APPROVED** [review]
Looks great

**Pull request merged**